src/test/render.tsx

```diff
@@ -15,7 +15,7 @@ import { SMDatasourceProvider } from 'contexts/SMDatasourceContext';
 import { getQueryClient } from 'data/queryClient';
 import { FeatureFlagProvider } from 'components/FeatureFlagProvider';
 
-import { TestRouteInfo } from '../routes/test/TestRouteInfo';
+import { TestRouteInfo } from './mocks/routing/TestRouteInfo';
```

Comment:
1. `TestRouteInfo` is not a mock (helper?)
2. Feels like the "owner" should be `customRender` (aka `render`) not `routing` 🤷🏻